packages/office-ui-fabric-react/src/components/Modal/Modal.styles.ts

```diff
@@ -83,6 +84,7 @@ export const getStyles = (props: IModalStyleProps): IModalStyles => {
       scrollableContentClassName
     ],
     layer: isModeless && [
+      layerProps && layerProps.className,
```

Comment:
actually I'm confused why you have to do this. The classname SHOULD be applied by the layer if the layerprops are being passed along.

Reply:
@dzearing Is your recommendation to remove `layerProps && layerProps.className,` and so avoid passing in the `layerProps` as you mention [above](https://github.com/OfficeDev/office-ui-fabric-react/pull/8273#discussion_r271009707)?

Reply:
> I believe this is needed so that `layerProps.className` will continue to get passed down to the layer while also adding the needed styling (below). Without this the `layerProps.className` will be completely overridden by the below styling when modeless
> 
> In reply to: [271020723](https://github.com/OfficeDev/office-ui-fabric-react/pull/8273#discussion_r271020723) [](ancestors = 271020723)

@dzearing, if we do keep the `layerProps && layerProps.className`, then, we need the `layerProps` prop. Please advise on this, and if you have any other feedback. 

@JasonGore if you had any other comments too.

Reply:
@dzearing does this resolve your comment? Please let me know how we can complete this PR.